pallet/staking/tests/mock.rs

```diff
@@ -75,6 +75,7 @@ impl pallet_assets::Config for Runtime {
 	type AssetAccountDeposit = ();
 	type AssetDeposit = ();
 	type AssetId = AssetId;
+	type AssetIdParameter = u64;
```

Comment:
Why not `AssetId` instead of `u64`?  If we want to update the id type afterwards, modifying one place is enough.